Classes/TMDebugConsole.m

```diff
@@ -269,25 +259,24 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 
 - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
-    DDLogMessage *message = self.messages[self.messages.count - indexPath.row - 1];
-    NSString *messageText = [self textForMessage:message];
-    CGSize maximumMessageSize = CGSizeMake(self.tableView.bounds.size.width - kTMDebugConsoleTableViewCellHorizontalPadding, CGFLOAT_MAX);
-    CGSize messageSize = [messageText boundingRectWithSize:maximumMessageSize
-                                                   options:NSStringDrawingUsesLineFragmentOrigin
-                                                attributes:@{NSFontAttributeName: self.fontForMessages}
-                                                   context:nil].size;
-    
+    TMDebugConsoleMessage *message = self.messages[self.messages.count - indexPath.row - 1];
+	NSString *messageText = [self textForMessage:message];
+	CGSize maximumMessageSize = CGSizeMake(self.tableView.bounds.size.width - kTMDebugConsoleTableViewCellHorizontalPadding, CGFLOAT_MAX);
+	CGSize messageSize = [messageText boundingRectWithSize:maximumMessageSize
+												   options:NSStringDrawingUsesLineFragmentOrigin
+												attributes:@{NSFontAttributeName: self.fontForMessages}
+												   context:nil].size;
+
     return messageSize.height + kTMDebugConsoleTableViewCellVerticalSpacing;
 }
 
 #pragma mark - Message-dependent properties
 
-- (NSString *)textForMessage:(DDLogMessage *)message
+- (NSString *)textForMessage:(TMDebugConsoleMessage *)message
 {
-    NSMutableString *logMessage = [message->logMsg mutableCopy];
-	CFStringTrimWhitespace((CFMutableStringRef)logMessage);
-
-	NSString *formattedTime = [self.dateFormatter stringFromDate:message->timestamp];
+    NSMutableString *logMessage = [message->_logMessage mutableCopy];
```

Comment:
Probably it would be better to use the property instead of accessing the ivar ;)